bcda/bcdacli/cli_test.go

```diff
@@ -668,7 +668,7 @@ func (s *CLITestSuite) TestImportCCLFDirectory() {
 	assert.Nil(err)
 	assert.Contains(buf.String(), "Completed CCLF import.")
 	assert.Contains(buf.String(), "Successfully imported 2 files.")
-	assert.Contains(buf.String(), "Skipped 4 files.")
+	assert.Contains(buf.String(), "Skipped 5 files.")
```

Comment:
Changed to 5 with the introduction of a valid foldername, but unsupported ACO (T.UNKNOWNACO.ACO.ZC0Y18.D181120.T1000010)